app/models/bid.rb

```diff
@@ -1,2 +1,4 @@
 class Bid < ActiveRecord::Base
+  belongs_to :user
```

Comment:
Will we want to have a user only be able to bid on an item if it doesn't belong to them?